mitosheet/mitosheet/tests/step_performers/test_filter.py

```diff
@@ -259,6 +260,12 @@
         "2",
         pd.DataFrame(data={"A": ["12", "122222"]}),
     ),
+    (
+        pd.DataFrame(data={"A": ["abcDEF", "ABCdef", "def", "dEf"]}),
+        FC_STRING_CONTAINS_CASE_INSENSITIVE,
+        "aBc",
+        pd.DataFrame(data={"A": ["abcDEF", "ABCdef"]}),
+    ),
```

Comment:
Two more tests that might be nice: 
1. Combining two contains case insensitive with an And
2. Combinion two contains case insensitive with an Or

Just so we cover all our cases here and make sure they are working properly!